binding-parent/datastore-specific-descriptor/src/main/resources/assemblies/datastore-specific-assembly.xml

```diff
@@ -0,0 +1,57 @@
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+  <id>dist</id>
+  <includeBaseDirectory>true</includeBaseDirectory>
+  <baseDirectory>ycsb-${artifactId}-${version}</baseDirectory>
+  <files>
+    <file>
+      <source>README.md</source>
+      <outputDirectory></outputDirectory>
+    </file>
+  </files>
+  <fileSets>
+    <fileSet>
+      <directory>..</directory>
+      <outputDirectory></outputDirectory>
+      <fileMode>0644</fileMode>
+      <includes>
+        <include>CHANGELOG</include>
+        <include>LICENSE.txt</include>
+        <include>NOTICE.txt</include>
+      </includes>
+    </fileSet>
+    <fileSet>
+      <directory>../bin</directory>
```

Comment:
sure, I could do that. I'd guess this was a left over from when we had the two scripts. Can always switch back if we end up with more things in the directory.

I'd like it even more to remove the top level bin and keep the ycsb script in src/main/python or some such and only have bin made in the distro, but I think folks rely on being able to run directly from source check out too often right now. :/